libbeat/docs/config-file-format.asciidoc

```diff
@@ -0,0 +1,355 @@
+[[config-file-format]]
+== Config file format
+
+Beats config files are based on http://www.yaml.org[YAML], a file format that is
+easier to read and write then other common data formats like XML or JSON.
+
+In beats all YAML files start with a "dictionary" (often called a "hash"). In
```

Comment:
Since you introduce "dictionary" as a new term you should provide a short definition.